STYLE_API_TEMPLATE.md

```diff
@@ -0,0 +1,142 @@
+* *Path parameter* - "Path parameter" and "URL parameter" are sometimes used synonymously. To avoid confusion, we use "path parameter" in this documentation.
+* *Query parameter* - This parameter name is often used synonymously with "request parameter." We use "query parameter" to be consistent.
+
+
+### General usage for code elements
+
+When you describe any code element such as an API, a parameter, or a field in a sentence you can use the noun name. 
+  *Example usage*:
+  The time field provides a timestamp for job completion.
+
+When you provide an exact example with a value, you can use the code element in code font. 
+  *Example usage*: 
+  The response provides a value for ```time_field``` , such as “timestamp.” 
+
+See also [Examples](https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/examples.html) in the AWS Style Guide.
+
+## Basic elements for documentation
+The basic structure for API documentation is informed by the following elements, each discussed under their respective headings below.
+
+When formatting, follow the heading hierarchy presented here. Depending on where the documentation appears within a section or subsection, heading levels may be adjusted to fit with other content.
+
+1. Name of API (heading level 2)
+2. Path parameters (heading level 3)
+3. Query parameters (heading level 3)
+4. Sample request (heading level 4)
+5. Sample response (heading level 4)
+6. Request fields (heading level 3)
+7. Sample request (heading level 4)
+
+## API name
+
+Provide an API name that describes its function, followed by a description of its top use case and any usage recommendations.
+
+*Example function*: "Autocomplete queries"
+
+Use sentence capitalization for the heading (for example, "Create or update mappings"). When you refer to the API operation, you can use lowercase with code font.
+
+If there is a corresponding OpenSearch Dashboards feature, provide a “See also” link that references it. 
+*Example*:  “To learn more about monitor findings, see [Document findings](https://opensearch.org/docs/latest/monitoring-plugins/alerting/monitors/#document-findings)."
+
+If applicable, provide any caveats to its usage with a note or tip, as in the following example:
+
+"If you use the security plugin, make sure you have the appropriate permissions."
+(To set this point in note-style format, follow the text on the next line with {: .note })
+
+### Path parameters
+
+While the API endpoint states a point of entry to a resource, the path parameter identifies a specific resource within it. Path parameters are located in the path of the endpoint just before the query string and following the resource name in the URL.
+
+
+```json
+GET _search/scroll/<scroll_id>
+```
+In the example above, the endpoint is "scroll" and its path parameter is "scroll_id".
```

Comment:
I like tickmarks for contrast when naming parameters. endpoint is `scroll` and its path parameter is `<scroll id>`. Triangular brackets signify that the scroll id will be replaced with an actual integer.